docs/DeveloperGuide.adoc

```diff
@@ -732,9 +732,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
 
-{More to be added}
+|`* * *` |new user |see suggested commands as I type them |easily remember what to write without referring to the help sheet
+
+|`* * *` |user |edit information about a contact without retyping everything |save time and work
```

Comment:
Here can be a little more specific, `without retyping everything` is a bit too ambiguous.
You can probably just mention, edit a specific field of the contact so that I can update information

Reply:
@tby1994